mmv1/third_party/terraform/services/compute/resource_compute_security_policy.go.tmpl

```diff
@@ -326,7 +326,6 @@ func ResourceComputeSecurityPolicy() *schema.Resource {
 										Description: `Rate limit key name applicable only for the following key types: HTTP_HEADER -- Name of the HTTP header whose value is taken as the key value. HTTP_COOKIE -- Name of the HTTP cookie whose value is taken as the key value.`,
 									},
 
-									{{ if ne $.TargetVersionName `ga` -}}
 									"enforce_on_key_configs": {
```

Comment:
We'll probably also need to remove the version guard in the expander and flattener at the following places:
https://github.com/maxi-cit/magic-modules/blob/385c4194a932fab82cb02db8c2db294ae9a83e89/mmv1/third_party/terraform/services/compute/resource_compute_security_policy.go.tmpl#L1481

https://github.com/maxi-cit/magic-modules/blob/385c4194a932fab82cb02db8c2db294ae9a83e89/mmv1/third_party/terraform/services/compute/resource_compute_security_policy.go.tmpl#L1506

https://github.com/maxi-cit/magic-modules/blob/385c4194a932fab82cb02db8c2db294ae9a83e89/mmv1/third_party/terraform/services/compute/resource_compute_security_policy.go.tmpl#L1539

https://github.com/maxi-cit/magic-modules/blob/385c4194a932fab82cb02db8c2db294ae9a83e89/mmv1/third_party/terraform/services/compute/resource_compute_security_policy.go.tmpl#L1574

Reply:
Hey there, you re totally right. I forget about those.